core/drivers/gic.c

```diff
@@ -267,6 +277,32 @@ static void gic_it_disable(struct gic_data *gd, size_t it)
 	write32(mask, gd->gicd_base + GICD_ICENABLER(idx));
 }
 
+static void gic_it_set_pending(struct gic_data *gd, size_t it)
+{
+	size_t idx = it / NUM_INTS_PER_REG;
+	uint32_t mask = 1 << (it % NUM_INTS_PER_REG);
```

Comment:
`gic_it_disable()` was written before we introduced this macro.
Please change to `BIT32(it % NUM_INTS_PER_REG)` here, we can deal with `gic_it_disable()` later.
We should in general replace all these `1 << x` with `BIT(x)` or `BIT32(x)` when convenient, but in separate patches to avoid making real changes harder to read.